common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -313,6 +317,52 @@ def test_from_xml(self):
             'transcripts': {'uk': 'ukrainian_translation.srt', 'de': 'german_translation.srt'},
         })
 
+    @ddt.data('course-v1:test_course+test_num+test_run', 'test_course/test_num/test_run')
+    def test_from_xml_when_handout_is_course_assest(self, course_id_string):
+        """
+        Test that if handout link is course_assest then it will contain targeted course_id in handout link.
+        """
+        module_system = DummySystem(load_error_modules=True)
+        asset_path = 'test.png'
+        course_id = CourseKey.from_string(course_id_string)
+        xml_data = '''
+            <video display_name="Test Video"
+                   youtube="1.0:p2Q6BrNhdh8,0.75:izygArpw-Qo,1.25:1EeWXzPdhSA,1.5:rABDYkeK0x8"
+                   show_captions="false"
+                   download_track="false"
+                   start_time="00:00:01"
+                   download_video="false"
+                   end_time="00:01:00">
+              <source src="http://www.example.com/source.mp4"/>
+              <track src="http://www.example.com/track"/>
+              <handout src="/asset-v1:test_course_1+test_num_1+test_run_1+type@asset+block@test.png"/>
+              <transcript language="uk" src="ukrainian_translation.srt" />
+              <transcript language="de" src="german_translation.srt" />
+            </video>
+        '''
+        id_generator = Mock()
+        id_generator.target_course_id = course_id
+        expected_handout_link = StaticContent.compute_location(course_id, asset_path)
+        expected_handout_link = StaticContent.serialize_asset_key_with_slash(expected_handout_link)
```

Comment:
hardcode what `expected_handout_link` is? Otherwise the test doesn't really test anything